install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -356,3 +371,20 @@ rules:
       - get
       - list
       - watch
+
+---
+# Roles needed by cluster-api in order to read openshift-config:cloud-provider-config configmap
```

Comment:
nit: replace cluster-api with machine-api

Reply:
can you add a comment referencing where this is coming from i.e https://github.com/openshift/installer/blob/a76a6b34092d99835d0e947ae9343d231f38f7e1/pkg/asset/manifests/cloudproviderconfig.go#L80

I'm not totally sure it's fair to assume this name or should be inferred from https://github.com/openshift/installer/blob/master/pkg/asset/manifests/infrastructure.go#L167